src/pages/authenticated/composer/_components/ControlPanel.tsx

```diff
@@ -112,7 +112,8 @@ export default (props: ControlPanelProps) => {
                   mkProgram={props.mkProgram}
                   mkOperator={props.mkOperator}
                   isAdvancedSettingsOpen={false}
-                  displayFields={{ program: false }}
+                  jobType={props.jobType as JobsJobType}
```

Comment:
I think the import statement for the `JobsJobType` type is missing, which is causing a compile-time error.
Please check the build results for more details: 
[https://ap-northeast-1.console.aws.amazon.com/amplify/apps/d214gyo0fl3unu/branches/fix-estimation-job-submmit/deployments](https://ap-northeast-1.console.aws.amazon.com/amplify/apps/d214gyo0fl3unu/branches/fix-estimation-job-submmit/deployments)